TeensyDmx.cpp

```diff
@@ -892,7 +994,514 @@ void TeensyDmx::maybeIncrementChecksumFail()
     }
 }
 
-void TeensyDmx::processRDM()
+
+void TeensyDmx::doRDMDiscovery() {
+    m_uidCount = 0;
+    m_dubPointer = 0;
+    m_discoveryState = DiscoveryState::DISCOVERY_UN_MUTE;
+    // Action queued, do it ASAP
+    m_nextDiscoveryAction = 0;
+    m_dubLowerBoundUid = RDM_MIN_LOWER_BOUND_UID;
+    m_dubUpperBoundUid = RDM_MAX_UPPER_BOUND_UID;
+    m_dubQueue[(m_dubPointer * 2)] = m_dubLowerBoundUid;
```

Comment:
m_dubPointer is always 0 here, so this is not required.

Reply:
True, although I'd hope it would be optimised out? I'd probably like to leave it as is for the moment, until I'm happy the code has settled down a bit more.